src/components/App/components/UserInput/UserInput.js

```diff
@@ -68,15 +70,22 @@ function UserInput(props) {
                 inputProps={{
                   id: 'font-size',
                   type: 'number',
-                  min: MIN_FONT_SIZE,
-                  max: MAX_FONT_SIZE,
+                  min: MIN_FONT_SIZE_IN_PX,
+                  max: MAX_FONT_SIZE_IN_PX,
```

Comment:
What if we don't constrain the font size in the Preview, so the preview font size could be in the range 8..48px (which is 36pt).




What if we don't constrain the font size in the Preview, so it could range 
from: 8px
to: